sandbox/build.gradle

```diff
@@ -63,8 +63,10 @@ dependencies {
     implementation deps.external.mviCoreAndroid
     implementation deps.external.rxrelay2
     implementation deps.external.rxandroid2
+    implementation deps.external.leakCanary // use debugImplementation in real life apps
```

Comment:
maybe let's use debugImplemenatation in sandbox as well to showcase how to configure it?

Reply:
#217